### DISCUSSION TITLE
Create Account issue when using three digits country codes like Libya

### DISCUSSION THREAD

**Discussion by NavinVr** [community/community#89433] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,

I'm using linphone sdk (org.linphone:linphone-sdk-android:5.2.1) in our application.
When we are creating a new account in the app, the user enter their number and selects country code.
In most of the cases, everything works fine but when the user selects a three digit country code like (Libya - +218).
Then the first digit of the number gets omitted automatically from the SDK side.

Before passing the number and country code in "accountCreator.setPhoneNumber()", the number is fine. But after this when we receive the number using accountCreator.getPhoneNumber(). The number first digit is omitted.

For example, I'm passing number - 931367399 and country code - +218 in "accountCreator.setPhoneNumber()" and the response which I get from accountCreator.getPhoneNumber() is +21831367399.

This is happening on all the android devices.

Device: Pixel 3
OS: Android 12
Version of the SDK - 5.2.1
Where you did got it from - Github
Android OS
Please let me know if anyone is facing same issue and how to fix this.

Thank you,

**Comment by AndreaGriffiths11** [comment]
From your description, it appears the issue is linked to the SDK not GitHub. Maybe open an issue in the SDK's repo & see what other documentation they have. You might find some helpful info on handling three-digit country codes. Good luck 🍀
